Source/ExcelDnaPack/ResourceHelper.cs

```diff
@@ -167,7 +178,7 @@ public void RemoveResource(string typeName, string name)
         {
             lock (lockResource)
             {
-                bool result = ResourceHelper.UpdateResource(_hUpdate, typeName, name, localeEnglishUS, IntPtr.Zero, 0);
+                bool result = ResourceHelper.UpdateResource(_hUpdate, typeName, name, localeNeutral, IntPtr.Zero, 0);
```

Comment:
nice that you also fixes this. I had already seen this, but didn't want to add another building lot